README.md

```diff
@@ -1,2 +1,70 @@
 # writefreely_ynh
-WriteFreely is a beautifully pared-down blogging platform that's simple on the surface, yet powerful underneath
+
+[![Integration level](https://dash.yunohost.org/integration/writefreely.svg)](https://ci-apps.yunohost.org/jenkins/job/writefreely%20%28Community%29/lastBuild/consoleFull)  
+[![Install WriteFreely with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=writefreely)
+
+> *This package allow you to install WriteFreely quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.*
+
+## Overview
+WriteFreely is a beautifully pared-down blogging platform that's simple on the surface, yet powerful underneath.
+
+Can be run as Single User Blog or Multi User Instance.
+
+Each User canbe limited from 1 to unlimited blogs.
+
+**Shipped version:** 0.70
+
+## Screenshots
+
+![](Link to an screenshot for this app)
```

Comment:
The link is not working. The pic is not seen.

Reply:
I have updated the pull request removing the screenshot part (As I don't know for now how to store a screenshot on GitHub)

I have also updated the backup, restore and upgrade steps.

I have done some tests with package_check
The only issue is: writefreely doesn't allow the username "package_checker"
Forcing the admin name, here is the resulte of package checker:

```
Package linter:                  SUCCESS
Installation:                    SUCCESS
Deleting:                        SUCCESS
Installation in a sub path:      Not evaluated.
Deleting from a sub path:        Not evaluated.
Installation on the root:        SUCCESS
Deleting from root:              SUCCESS
Upgrade:                         SUCCESS
Installation in private mode:    SUCCESS
Installation in public mode:     SUCCESS
Multi-instance installations:    Not evaluated.
Malformed path:                  SUCCESS
Port already used:               Not evaluated.
Backup:                          SUCCESS
Restore:                         SUCCESS
Change URL:                      Not evaluated.
This app doesn't respect the YEP 1.7 ! (https://yunohost.org/#/packaging_apps_guidelines_fr)
grep: /root/package_check/levels.json: No such file or directory
Level of this application: 3 ()
           Level 1: 1
           Level 2: 1
           Level 3: 1
           Level 4: 0
           Level 5: 1
           Level 6: 0
           Level 7: 1
           Level 8: 0
           Level 9: 0
           Level 10: 0
You can find the complete log of these tests in /root/package_check/Complete.log
```